cdap-kafka/src/main/java/co/cask/cdap/kafka/run/KafkaServerMain.java

```diff
@@ -81,6 +81,27 @@ public void init(String[] args) {
                                          KafkaConstants.DEFAULT_REPLICATION_FACTOR);
     LOG.info("Using replication factor {}", replicationFactor);
 
+    int retentionHours = cConf.getInt(KafkaConstants.ConfigKeys.RETENTION_HOURS_CONFIG,
+                                      KafkaConstants.DEFAULT_RETENTION_HOURS);
+
+    int bufferSendBytes = cConf.getInt(KafkaConstants.ConfigKeys.BUFFER_BYTES_SEND_CONFIG,
+                                       KafkaConstants.DEFAULT_BUFFER_BYTES);
+    int bufferReceiveBytes = cConf.getInt(KafkaConstants.ConfigKeys.BUFFER_BYTES_RECEIVE_CONFIG,
+                                          KafkaConstants.DEFAULT_BUFFER_BYTES);
+
+    int socketRequestMaxBytes = cConf.getInt(KafkaConstants.ConfigKeys.MAX_BYTES_CONFIG,
+                                             KafkaConstants.DEFAULT_SOCKET_MAX_BYTES);
+
+    long intervalMessageSize = cConf.getLong(KafkaConstants.ConfigKeys.INTERVAL_MESSAGES_CONFIG,
+                                             KafkaConstants.DEFAULT_INTERVAL_MESSAGE_SIZE);
+
+    long messagePersistTime = cConf.getLong(KafkaConstants.ConfigKeys.INTERVAL_MS_CONFIG);
```

Comment:
Oh i was looking at the Kafka docs for default values and it said "null", but I now that I look back, it can (and i guess probably should) be 1000 like we originally set it to be.